tjenestespesifikasjoner/pdl-api/src/main/resources/pdl/queries/hentPersondata.graphql

```diff
@@ -190,6 +190,12 @@ query($ident: ID!){
             }
         }
         bostedsadresse {
+            metadata {
+                endringer {
+                    registrert
+                    registrertAv
```

Comment:
Kunne vi gjort noe alà;

```
fragment sistEndret on Metadata {
    endringer {
        registrert
        registrertAv
        systemkilde
    }
}
```
og deretter brukt det sånn;
```
bostedsadresse {
  metadata { ...sistEndret }
}
```

På denne måten så kan vi få samlet håndtering av sist-endret på tvers av informasjonselementene i spørringen. F.eks så kan vi se for oss at kontaktadresse skal ha samme type oppsett, og telefon har allerede ett konsept om sist endret.